src/v/kafka/server/handlers/describe_configs.cc

```diff
@@ -77,12 +78,42 @@ static ss::sstring describe_as_string(const T& t) {
     return ssx::sformat("{}", t);
 }
 
+
+// property_config_type maps the datatype for a configuration to describe_configs_type.
+// Currently class_type and password are not used in Redpanda so we do not include checks for those types.
+// You may find a similar mapping in Apache Kafka at https://github.com/apache/kafka/blob/be032735b39360df1a6de1a7feea8b4336e5bcc0/core/src/main/scala/kafka/server/ConfigHelper.scala
+// Type sizes are based on Java and taken from https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html
+template<typename T>
+describe_configs_type property_config_type() {
```

Comment:
you can probably make this function `consteval`

Reply:
I had to remove `consteval` so I can check the property name for types such as `delete_retention_ms`